apps/omg_watcher_info/test/omg_watcher_info/block_applicator_test.exs

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019-2020 OMG Network Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.BlockApplicatorTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: true
```

Comment:
```suggestion
  use ExUnit.Case, async: false
```
needs to be false when using ecto. cc @pgebal

Reply:
https://github.com/omgnetwork/elixir-omg/pull/1769/commits/028b4bea38c5ae0d8258bf1e850173cea6ebc7c5